monitor.go

```diff
@@ -13,8 +13,13 @@ func main() {
 		log.Fatal(err)
 	}
 
-	listener := make(chan *docker.APIEvents)
-	err = cli.AddEventListener(listener)
+	listener := make(chan *docker.APIEvents, 10)
+	filters := map[string][]string{
+		"type":  {"container"},
+		"event": {"create"},
+	}
+	opts := docker.EventsOptions{Filters: filters}
+	err = cli.AddEventListenerWithOptions(opts, listener)
```

Comment:
While reading the docs, I noticed we could _prefilter_ the events to receive, instead of receiving everything and doing manual filtering.